### PR TITLE
Redwall

### PR DESCRIPTION
**Pull request by depperm**
Redwall series by Brian Jacques

### CODE REVIEW

**Comment by RichardLitt** [comment]
Added a note on food.

**Comment by RichardLitt** [comment]
Thank you!

File: contributing.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Please ensure your pull request adheres to the following guidelines:
 - Use the following format:
 
 ```
-[title](link) (year) _by [author](link)_ [rating]
+#### [title](link) (year) _by [author](link)_ [rating]
 
 Commentary - [@you](https://github.com/you)
```
Comment (0367971847) [inline contributing.md:15]: Hi

**Review by 0367971847 — COMMENTED** [review]
Bye

**Pull request merged**